bundle/internal/tf/schema/resource_job.go

```diff
@@ -1396,11 +1396,17 @@ type ResourceJobTriggerTableUpdate struct {
 	WaitAfterLastChangeSeconds    int      `json:"wait_after_last_change_seconds,omitempty"`
 }
 
+type ResourceJobTriggerPeriodic struct {
```

Comment:
This code is auto-generated from Databricks Terraform provider schema and should not be edited manually.  Instead please follow https://github.com/databricks/cli/tree/main/bundle/internal/tf/codegen

Reply:
This reason why it might not auto-generate now is that your changes for period was not added in TF schema here
https://github.com/databricks/terraform-provider-databricks/blob/154ea38202a53d2c406904f6dcb6c31524956034/jobs/resource_job.go#L517

Reply:
No, it does not happen automatically, we run the changes manually when upgrading TF provider to a new version.

CLI is already upgraded to the latest version of TF provider here https://github.com/databricks/cli/pull/1626 and the changes were auto-generated, so not entirely sure where you diff comes through